boards/seagate/legend/legend.dts

```diff
@@ -22,6 +22,7 @@
 	aliases {
 		watchdog0 = &iwdg;
 		spi-flash0 = &spi_nor;
+		led-strip = &led_strip_spi;
```

Comment:
I am not sure... This alias is not missing. It is provided in the `legend_*.overlay` DT files. But since it's used on all revisions of the device, I don't know why it wasn't shared in `legend.dts` in the first place...

Reply:
It was a half-hearted fix, so I fixed it.
Even if we do not modify this, it will not affect the build, but I think it is better to make it common.

Reply:
Please fix the name of my employer in the commit title :)

seagage -> seagate

Reply:
Oops, I'm very sorry about that. fixed.